### PR TITLE
Add support for the embedded source code

### PR DESCRIPTION
**Pull request by hliatis**
These changes get the source code from OpSource and OpSourceContinued

### CODE REVIEW

File: spirv_reflect.c [hunk 1]
```diff
@@ -645,6 +647,32 @@ static SpvReflectResult ParseNodes(Parser* p_parser)
         if (p_node->word_count >= 4) {
           CHECKED_READU32(p_parser, p_node->word_offset + 3, p_parser->source_file_id);
         }
+        if (p_node->word_count >= 5) {
+          const char* p_source = (const char*)(p_parser->spirv_code + p_node->word_offset + 4);
+          char* p_source_temp = (char*)calloc(strlen(p_source), sizeof(char*));
```
Comment (chaoticbob) [inline spirv_reflect.c:652]: Is it possible for `p_source` to point to an empty string resulting in `strlen` returning 0?

Comment (hliatis) [reply spirv_reflect.c:652]: I'm not 100% sure. I assumed that if there is a 5th word, then the source code string exists. However, I think I am missing a +1 here.

Comment (chaoticbob) [reply spirv_reflect.c:652]: IIRC `strcpy` below will copy with the null terminator?

Comment (hliatis) [reply spirv_reflect.c]: +1 added

Comment (chaoticbob) [reply spirv_reflect.c]: Looks good, Thanks.

**Pull request merged**